src/main/java/com/uid2/admin/vertx/service/AdminKeyService.java

```diff
@@ -124,12 +124,34 @@ public void setupRoutes(Router router) {
         }), Role.ADMINISTRATOR));
     }
 
+    @Override
+    public Collection<OperationModel> qldbSetup(){
+        try {
+            Collection<AdminUser> adminUsers = adminUserProvider.getAll();
+            Collection<OperationModel> newModels = new HashSet<>();
+            for (AdminUser a : adminUsers) {
+                newModels.add(new OperationModel(Type.ADMIN, a.getName(), null,
+                        DigestUtils.sha256Hex(jsonWriter.writeValueAsString(a)), null));
+            }
+            return newModels;
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
If this fails will it cause admin to fail because logging isn't setup correctly?

Reply:
I'll come around to address this case, but this falls under the category of failure cases which I'm planning to handle along with some other things in a future PR. For now I just want to see if I can get the integ environment up and running at all